is.go

```diff
@@ -0,0 +1,80 @@
+// SPDX-FileCopyrightText: 2025 Comcast Cable Communications Management, LLC
+// SPDX-License-Identifier: Apache-2.0
+
+package wrp
+
+// Is reports whether the msg is the same type as the target, or is convertible
+// to the target.
+//
+// If the msg parameter is a *Message, it will be validated against the provided
+// validators.  If the msg is not a *Message, the validators will be ignored.
+// If the validators are not provided, the msg will be validated against the
+// default validators.  To skip validation, provide the NoStandardValidation()
+// as a validator.
+func Is(msg, target Union, validators ...Processor) bool {
```

Comment:
It's a bit odd to ignore the `validators` based on the type of `msg`.  What you might do is make `Processor` generic in terms of the `msg` type:

```
func Is[S, T Union](msg S, target T, validators ... Processor[S]) bool
```

If that's not ok, then I would have two variants of `Is` to make it clearer:

```
func Is(msg, target Union) bool

func IsValidate(msg *wrp.Message, target Union, validators ...Processor) bool
```

Reply:
That's a good point.  I'll fix this either by making more like you'd expect or split them out.